libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -738,10 +738,10 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
 
     // @Param: OPTIONS
     // @DisplayName: Optional EKF behaviour
-    // @Description: This controls optional EKF behaviour. Setting JammingExpected will change the EKF nehaviour such that if dead reckoning navigation is possible it will require the preflight alignment GPS quality checks controlled by EK3_GPS_CHECK and EK3_CHECK_SCALE to pass before resuming GPS use if GPS lock is lost for more than 2 seconds to prevent bad
-    // @Bitmask: 0:JammingExpected
+    // @Description: This controls optional EKF behaviour. Setting JammingExpected will change the EKF behaviour such that if dead reckoning navigation is possible it will require the preflight alignment GPS quality checks controlled by EK3_GPS_CHECK and EK3_CHECK_SCALE to pass before resuming GPS use if GPS lock is lost for more than 2 seconds to prevent bad GPS measurements from getting fused. If PlaneRequiresGPS is true, EKF3 will not initialize unless the preferred GPS has at least GPS_OK_FIX_3D.
+    // @Bitmask: 0:JammingExpected,1:PlaneRequiresGPS
     // @User: Advanced
-    AP_GROUPINFO("OPTIONS",  11, NavEKF3, _options, 0),
+    AP_GROUPINFO("OPTIONS",  11, NavEKF3, _options, 2),
```

Comment:
We can't really change the default on a bitmask, sorry.  If someone has set any other bits in their options then this new bit wouldn't be set in their parameter and thus their vehicle would change behaviour underneath them.

This has to be inverted, so something like `1:SuppressGPSFixCheck`

Reply:
Gotcha, o.k. will do.